interpreter/valid/valid.ml

```diff
@@ -515,14 +526,15 @@ let check_export (c : context) (set : NameSet.t) (ex : export) : NameSet.t =
   | TableExport x -> ignore (table c x)
   | MemoryExport x -> ignore (memory c x)
   | GlobalExport x -> ignore (global c x)
+  | ExceptionExport x -> ignore (exception_ c x)
```

Comment:
`c.exceptions` seems to be always empty, thus every export fails with an error along the lines of
```
../test/core/exception.wast:6.3-6.44: invalid module: unknown exception 2
```

Reply:
Nevermind. I noticed that I never populated it. The latest commit rectifies this mistake.